vslib/src/TrafficForwarder.cpp

```diff
@@ -0,0 +1,100 @@
+#include "TrafficForwarder.h"
+
+#include "swss/logger.h"
+
+#include <memory.h>
+
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <net/if.h>
+#include <linux/if_tun.h>
+#include <sys/ioctl.h>
+#include <net/if_arp.h>
+#include <unistd.h>
+#include <net/ethernet.h>
+#include <arpa/inet.h>
+#include <linux/if_packet.h>
+#include <linux/if_ether.h>
+
+using namespace saivs;
+
+#define ETH_FRAME_BUFFER_SIZE (0x4000)
```

Comment:
is this redefined in many places ?

Reply:
Done, I move it to the header. Please check it.